.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2.1
+workflows:
+  main:
+    jobs:
+      - test
+      - release:
+          requires:
+            - test
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/          
+jobs:
+  test:
+    docker:
+      - image: docker.io/golang:1.14.0
+    steps:
+      - checkout
+      - run:
+          name: Test
+          command: make test
+  release:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Prepare env
+          command: |
+            echo ${DOCKERHUB_SECRET} | docker login -u ${DOCKERHUB_USER} --password-stdin
+      - run: curl -sL https://git.io/goreleaser | bash
```

Comment:
Maybe I remember it wrong but.. Does `goreleaser` need a GitHub token?

Do you plan to put the @poiana token into a CircleCI context right?

Reply:
Yes, `goreleaser` needs a valid `GITHUB_TOKEN` with the `repo` scope.
I would use the @poiana one, but I don't have access to it